src/lib.rs

```diff
@@ -3684,6 +3675,17 @@ impl Tool {
         self.family == ToolFamily::Clang
     }
 
+    /// Whether the tool is AppleClang under .xctoolchain
+    #[cfg(target_vendor = "apple")]
+    fn is_xctoolchain_clang(&self) -> bool {
+        let path = self.path.to_str().unwrap();
```

Comment:
This has bitten us before, it should probably be to_string_lossy.

Reply:
Thank you, I fixed it.